spec/source_file_spec.rb

```diff
@@ -4,7 +4,11 @@
 
 if SimpleCov.usable?
   describe SimpleCov::SourceFile do
-    COVERAGE_FOR_SAMPLE_RB = [nil, 1, 1, 1, nil, nil, 1, 0, nil, nil, nil, nil, nil, nil, nil, nil].freeze
+    COVERAGE_FOR_SAMPLE_RB = {
+      :lines => [nil, 1, 1, 1, nil, nil, 1, 0, nil, nil, nil, nil, nil, nil, nil, nil],
+      :branches => {[:if, 0, 17, 6, 23, 9] => {[:then, 1, 18, 8, 18, 81] => 3, [:else, 2, 20, 8, 22, 19] => 0}, [:if, 3, 29, 6, 35, 9] => {[:then, 4, 30, 8, 30, 81] => 3, [:else, 5, 32, 8, 34, 20] => 0}},
```

Comment:
Is there a way to indent this better? Maybe one line per key-value pair in value for `:branches`?